README.md

```diff
@@ -35,7 +37,7 @@ expect(subject).events(scheduler: scheduler, disposeBag: disposeBag)
         Recorded.next(5, "Hello"),
         Recorded.next(10, "World"),
         Recorded.completed(100)
-        ]))
+     ]))
```

Comment:
not sure if im imagining but it still looks unaligned

Reply:
Oh, now I get your idea, sorry :) 
Really strange case of formatting. I hope now it looks ok